Jenkinsfile.release

```diff
@@ -11,7 +11,7 @@ CQZ_VERSION = ""
 BUILD_STATUS_FAILED = 'Failed'
 BUILD_STATUS_SUCCESSFUL = 'Successful'
 
-EMAIL_LIST = 'alexander@cliqz.com,leonid@cliqz.com,lucian@cliqz.com'
```

Comment:
@luciancor  @alver-cliqz : May be a better idea to use a mailing list here instead of individual email IDs?

Reply:
good point, @alver-cliqz Marty can help with that ;)